openpype/hosts/traypublisher/plugins/create/create_editorial.py

```diff
@@ -0,0 +1,720 @@
+
+
+CLIP_ATTR_DEFS = [
+    EnumDef(
+        "fps",
+        items={
+            "from_selection": "From selection",
+            23.997: "23.976",
+            24: "24",
+            25: "25",
+            29.97: "29.97",
+            30: "30"
+        },
+        label="FPS"
+    ),
+    NumberDef(
+        "workfile_start_frame",
+        default=1001,
+        label="Workfile start frame"
+    ),
+    NumberDef(
+        "handle_start",
+        default=0,
+        label="Handle start"
+    ),
+    NumberDef(
+        "handle_end",
+        default=0,
+        label="Handle end"
+    )
+]
+
+
+class EditorialClipInstanceCreatorBase(HiddenTrayPublishCreator):
+    host_name = "traypublisher"
+
+    def __init__(
+        self, project_settings, *args, **kwargs
+    ):
+        super(EditorialClipInstanceCreatorBase, self).__init__(
+            project_settings, *args, **kwargs
+        )
+
+    def create(self, instance_data, source_data=None):
+        self.log.info(f"instance_data: {instance_data}")
+        subset_name = instance_data["subset"]
+
+        return self._create_instance(subset_name, instance_data)
+
+    def _create_instance(self, subset_name, data):
+
+        # Create new instance
+        new_instance = CreatedInstance(self.family, subset_name, data, self)
+        self.log.info(f"instance_data: {pformat(new_instance.data)}")
+
+        # Host implementation of storing metadata about instance
+        HostContext.add_instance(new_instance.data_to_store())
+        # Add instance to current context
+        self._add_instance_to_context(new_instance)
+
+        return new_instance
+
+    def get_instance_attr_defs(self):
+        return [
+            BoolDef(
+                "add_review_family",
+                default=True,
+                label="Review"
+            )
+        ]
+
+
+class EditorialShotInstanceCreator(EditorialClipInstanceCreatorBase):
+    identifier = "editorial_shot"
+    family = "shot"
+    label = "Editorial Shot"
+
+    def __init__(
+        self, project_settings, *args, **kwargs
+    ):
+        super(EditorialShotInstanceCreator, self).__init__(
+            project_settings, *args, **kwargs
+        )
+
+    def get_instance_attr_defs(self):
+        attr_defs = [
+            TextDef(
+                "asset_name",
+                label="Asset name",
+            )
+        ]
+        attr_defs.extend(CLIP_ATTR_DEFS)
+        return attr_defs
+
+class EditorialPlateInstanceCreator(EditorialClipInstanceCreatorBase):
+    identifier = "editorial_plate"
+    family = "plate"
+    label = "Editorial Plate"
+
+    def __init__(
+        self, project_settings, *args, **kwargs
+    ):
+        super(EditorialPlateInstanceCreator, self).__init__(
+            project_settings, *args, **kwargs
+        )
+
+
+class EditorialAudioInstanceCreator(EditorialClipInstanceCreatorBase):
+    identifier = "editorial_audio"
+    family = "audio"
+    label = "Editorial Audio"
+
+    def __init__(
```

Comment:
There is a lot of empty `__init__` methods. You can remove them...

Reply:
Isnt it important for super() method?

Reply:
but yeah, it works without them :)

Reply:
It is not.Calling just super method is the same as you would not override it at all :)